### DISCUSSION TITLE
/nsm storage filled, increase disk size doesn’t work

### DISCUSSION THREAD

**Discussion by cyber7878** [Security-Onion-Solutions/securityonion#7923] [Unsupported Versions] [answered]
Hey everyone, I have a problem
I’ve deployed security onion from ovf and the ovf set the storage to 100g 
When I deployed it I gave it a disk with size of 2t.
I’ve read that lvm volume is supposed to register the new disk space automatically and increase in size but it doesn’t do that for me.
The nsm partition filled up pretty quickly with just 58g.
I’ve tried to increase the disk itself with fdisk and it worked but the partitions didn’t change and are still full
I’ve also tried commands like vgextend and lvextend to make the partitions use the new disk size and it didn’t work
Can anyone help me increase the disk and partitions?

**Comment by dougburks** [comment, marked as answer]
> I’ve deployed security onion from ovf

We don't supply an OVF, so you might want to consider performing a fresh installation with our official ISO image and following the steps in our documentation:
https://docs.securityonion.net/en/2.3/getting-started.html

**Reply by cyber7878** [reply]
I install it from the iso in my testing environment and then add tools and changes of my own.
Then I export it to an ovf so than when I install it in production it will be a plug and play with all the settings and tools that I made.
Is there an option to increase disk size to nsm maybe? or to fix the problem that the lvm doesn’t register the disk space automatically?

**Reply by dougburks** [reply]
In theory, you should able to use standard LVM commands to take advantage of the additional space. But if you can't get that to work, then we'd suggest simply doing a direct installation of our ISO image as this should automatically take advantage of the space.
